src/basis/dom/event.js

```diff
@@ -165,6 +156,13 @@
           wheelDelta: wheelDelta(event)
         });
       }
+      else if (isTouchEvent(event))
+      {
+        extend(this, {
+          mouseX: touchX(event),
```

Comment:
`mouseX`/`mouseY` are using in several modules like `dragdrop.js`, `slider.js`, etc as base for various calculations, that's why we extend for example instance of `TouchEvent` by these non-native properties to provide common interface.
Should we replace them by `pointerX`/`pointerY` everywhere?